boomerang.js

```diff
@@ -3582,7 +3582,7 @@ BOOMR_check_doc_domain();
 					// remove this name from vars so it isn't also added
 					// to the non-prioritized list when pri=0 is called
 					if (pri !== 0) {
-						delete vars[name];
+						vars[name] = null;
```

Comment:
I don't understand why the `delete` is a problem here.  We're removing the `name` property from the `vars` object. Setting it to `null` has a different meaning since it will still show up in `Object.keys` and `hasOwnProperty`

Reply:
You're right, I changed it too hastily. Reverted.